closure/bin/build/jscompiler.py

```diff
@@ -69,7 +69,7 @@ def _JavaSupports32BitMode():
   return supported
 
 
-def _GetJsCompilerArgs(compiler_jar_path, java_version, source_paths,
+def _GetJsCompilerArgs(compiler_jar_path, java_version, source_paths, flags_file,
```

Comment:
We have an 80 char line length limit. I can't tell if this line or :133 are over 80.